internal/compiler/typeloader.rs

```diff
@@ -520,7 +518,7 @@ impl TypeLoader {
         &self,
         referencing_file: Option<&std::path::Path>,
         file_to_import: &str,
-    ) -> Option<(PathBuf, Option<&'static [u8]>)> {
+    ) -> Option<(std::path::PathBuf, Option<&'static [u8]>)> {
```

Comment:
Just curious about this.  Is it not `use`d anymore?

Reply:
Good catch, I'll fix this (and the others)